python/ittapi.native/pt_region.cpp

```diff
@@ -0,0 +1,272 @@
+#include "pt_region.hpp"
+
+#include <structmember.h>
+
+#include "string_handle.hpp"
+#include "extensions/string.hpp"
+
+namespace ittapi
+{
+namespace itt_NERSC
```

Comment:
PT regions are not NERSC implementation specific

Reply:
Have removed the itt_NERSC namespace in the latest committed changes([Link](https://github.com/intel/ittapi/pull/153/commits/e61caf110abf9a043480662a683aa8417d69bec9)). Please have a look.